DisguiseUnityRenderStream/Runtime/PluginEntry.cs

```diff
@@ -780,6 +784,7 @@ private PluginEntry()
                 Debug.LogError(string.Format("Failed to initialise: {0}", error));
             else
             {
+#if !UNITY_EDITOR
```

Comment:
A simple fix is to put graphics init logic in a `RuntimeInitializeOnLoadMethod` method.